### PR TITLE
wxGUI: Fixed E722 in ii2t_gis_set.py

### PR DESCRIPTION
**Pull request by arohanajit**
Fixed `E722: bare 'except'` in iit2t_gis_set.py. Added general catchall `Exception` for cases with multiple errors. Other were mostly `OSError and shutil.Error`. Also print error message in the error notification

### CODE REVIEW

**Comment by arohanajit** [comment]
Yeah, even though I cloned a fresh copy this morning. Normally I try to update the branch when reqd so that PR are in line with the latest version

**Comment by arohanajit** [comment]
Oh ok, got it. I do this to avoid any potential merge conflicts but will let it be in the future

File: gui/wxpython/image2target/ii2t_gis_set.py [hunk 1]
```diff
@@ -824,8 +824,8 @@ def DeleteMapset(self, event):
                 shutil.rmtree(os.path.join(self.gisdbase, location, mapset))
                 self.OnSelectLocation(None)
                 self.lbmapsets.SetSelection(0)
-            except:
-                wx.MessageBox(message=_("Unable to delete mapset"))
+            except (OSError, shutil.Error) as e:
```
Comment (petrasovaa) [inline gui/wxpython/image2target/ii2t_gis_set.py:827]: It doesn't look like shutil.Error is raised:
```
>>> try:
...     shutil.rmtree("/tmp/aaaa/aaa")
... except shutil.Error as e:
...     print(e)
... 
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/usr/lib/python3.12/shutil.py", line 775, in rmtree
    onexc(os.lstat, path, err)
  File "/usr/lib/python3.12/shutil.py", line 773, in rmtree
    orig_st = os.lstat(path, dir_fd=dir_fd)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/aaaa/aaa'
```

File: gui/wxpython/image2target/ii2t_manager.py [hunk 2]
```diff
@@ -895,7 +895,7 @@ def OnSrcSelection(self, event):
             if p.returncode == 0:
                 print("returncode = ", str(p.returncode))
                 self.parent.Map.region = self.parent.Map.GetRegion()
-        except:
```
Comment (petrasovaa) [inline gui/wxpython/image2target/ii2t_manager.py:898]: This is the same chunk of code we already saw, it's just copied over. You can remove the entire block.

File: gui/wxpython/image2target/ii2t_gis_set.py (hunk 1 above)
Comment (arohanajit) [reply gui/wxpython/image2target/ii2t_gis_set.py:827]: I'll add `FileNotFoundError` as well, which I think should cover pretty much all major errors `shutil.rmtree` can raise. Please let me know if you think a catch all `Exception` would be a better choice here

Comment (arohanajit) [reply gui/wxpython/image2target/ii2t_gis_set.py]: Got it, I'll only keep OSError in that case

**Review by petrasovaa — APPROVED** [review]

**Pull request merged**